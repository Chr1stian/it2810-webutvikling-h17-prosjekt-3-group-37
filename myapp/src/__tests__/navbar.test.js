import React from "react";
//Import Enzyme with shallow to test component as a unit
import Adapter from "enzyme-adapter-react-16";
import Enzyme from "enzyme";
Enzyme.configure({ adapter: new Adapter() });
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
//Importing component that will be tested
import NavBar from './../containers/NavBar.js'

//takes snapshot of component
test('NavBar component should render as expected', () => {
    const component = shallow(<NavBar />);
    const tree = toJson(component);
   //take snapshots to be noticed about changes tht might be unexpected
    expect(tree).toMatchSnapshot();
})
