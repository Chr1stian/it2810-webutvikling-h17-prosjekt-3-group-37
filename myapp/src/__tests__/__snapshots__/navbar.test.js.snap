// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar component should render as expected 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <ul
      className="App-navbar"
    >
      <li
        id="home"
      >
        <Link
          className="navLink"
          replace={false}
          to="/"
        >
          Home
        </Link>
      </li>
      <li
        id="todocontainer"
      >
        <Link
          className="navLink"
          replace={false}
          to="/todo"
        >
          Todo
        </Link>
      </li>
      <li
        id="notes"
      >
        <Link
          className="navLink"
          replace={false}
          to="/note"
        >
          Note
        </Link>
      </li>
      <li
        id="appointment"
      >
        <Link
          className="navLink"
          replace={false}
          to="/appointments"
        >
          Appointments
        </Link>
      </li>
    </ul>
  </header>
</div>
`;
